static/src/javascripts/projects/common/modules/crosswords/crossword.js

```diff
@@ -802,11 +807,13 @@ class Crossword extends Component {
                     clueInFocus={focused}
                     crossword={this}
                 />
+                <div class="below_controls"></div>
```

Comment:
I don't have a strong opinion on the exact name of this class, but we could probably use existing conventions e.g.
```suggestion
                <div class="crossword__container__below-controls"></div>
```